Zend/zend_compile.c

```diff
@@ -316,6 +316,7 @@ static bool zend_is_not_imported(zend_string *name) {
 void zend_oparray_context_begin(zend_oparray_context *prev_context) /* {{{ */
```

Comment:
I'd pass the zend_op_array as arg here to make is less easy to forget about setting it.